channel/player.go

```diff
@@ -160,6 +161,7 @@ func (d player) send(packet mpacket.Packet) {
 	}
 
 	d.conn.Send(packet)
+	d.save()
```

Comment:
Not a good idea to save to database every time a packet is sent as database saving is a slow operation. We should only save at very specific points to minimise the number of writes and the size of the update (currently the save function saves everything and needs to be broken down into categories such as inventory saves, character avatar saves, friends list saves, guild saves etc).

Reply:
That makes sense. I can remove this and work on adding specific saves.